modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectDetails/EditObjectDetails.tsx

```diff
@@ -145,13 +145,10 @@ export default function EditObjectDetails({
 				);
 
 				if (!publishResponse.ok) {
-					const {title} = (await publishResponse.json()) as {
-						status: string;
-						title: string;
-					};
-
 					openToast({
-						message: title,
+						message: Liferay.Language.get(
+							'object-definition-already-published'
```

Comment:
Hi Larissa, we'll need to change this key to adjust the message ok? Thanks :blush: 
```suggestion
							'the-object-definition-has-already-been-published'
```